### PR TITLE
Create 03-jasper-profile.md

### PR DESCRIPTION
**Pull request by JasperGilhuis**

### CODE REVIEW

**Comment by JasperGilhuis** [comment]
Added @jessehouwing to read along with my changes, I will notify you when I'm ready for review! 👍

File: docs/_slides/03-jasper-profile.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from [Jasper](https://github.com/jaspergilhuis)
+
+![avatar-jasper]([https://avatars.githubusercontent.com/u/4173387?v=4](https://octodex.github.com/images/privateinvestocat.jpg)
```
Comment (jessehouwing) [inline docs/_slides/03-jasper-profile.md:9]: Does that need to be a link to a picture to a link?

**Review by jessehouwing — CHANGES_REQUESTED** [review]

File: docs/_slides/03-jasper-profile.md (hunk 1 above)
Comment (JasperGilhuis) [reply docs/_slides/03-jasper-profile.md:9]: OK On it

Comment (JasperGilhuis) [reply docs/_slides/03-jasper-profile.md]: Fixed it can you have a look?

**Review by jessehouwing — APPROVED** [review]

**Pull request merged**